pkg/scalers/postgresql_scaler.go

```diff
@@ -213,7 +213,7 @@ func (s *postgreSQLScaler) GetMetrics(ctx context.Context, metricName string, me
 	}
 
 	metric := external_metrics.ExternalMetricValue{
-		MetricName: pgMetricName,
+		MetricName: metricName,
```

Comment:
`pgMetricName` variable is not used anymore, therefore you can remove the definition from line 18

Reply:
Done!